src/filesystem.ts

```diff
@@ -97,6 +101,46 @@ export async function loadFileSystem(
 }
 
 
+/**
+ * Create a filesystem
+ *
+ * @param permissions The permissions to initialize the filesystem
+ */
+export const createFilesystem = async (permissions: Permissions): Promise<FileSystem> => {
```

Comment:
Same comment as with the `useWnfs` naming, I think this function name might be confusing. The `loadFileSystem` function above actually can create a filesystem as well, but this function here does a few additional things.

There's a few use-cases in which you want to create a file system separate from your account's file system. This function would be confusing, since it directly overrides your account's filesystem. I think it should be called `assignNewSelfContainedFileSystem()` or something like that.

Reply:
What are the use cases where you want to create a filesystem separate from your account's filesystem? I see one case where we use it for testing. Are there others?

Perhaps `createFilesystem` could be modified to support the other use cases.

Reply:
The other main use case I had mind is when you have a temporary file system, ie. a file system for a user of the app that doesn't have a Fission account yet.

Maybe add an option to auto publish, so we can avoid accidental data loss situations? 🤷

Reply:
I gave this some thought, and I think that renaming this function is the way to go for now. What would you think about the name `initializeFilesystem`? Or maybe `setupFilesystem`? We can make its effects explicit in the documentation comment.

I'd prefer a less verbose name because developers will call this function directly after registering a user. At the point of registration, webnative will already be initialized, and we probably want to avoid requiring a reload to setup the filesystem.

Reply:
> I'd prefer a less verbose name because developers will call this function directly after registering a user. At the point of registration

Makes sense, in that case I'd pick `setupFileSystem`. But if I had to pick a single word, I'd probably pick something like `installFileSystem` or `establishFileSystem` (because 'setup' is too generic and too similar to 'load').

Reply:
Yep, agreed that setup is too generic. 👍 

Updated it to `installFileSystem` and added some comments to it. We can go with that for now and change it later if we come up with something else.

Reply:
Was chatting with @jessmartin about this. What would you think about `bootstrapFileSystem` instead of `installFileSystem`?

Reply:
> There's a few use-cases in which you want to create a file system separate from your account's file system. This function would be confusing, since it directly overrides your account's filesystem. I think it should be called assignNewSelfContainedFileSystem() or something like that.

Random suggestion from the process-based world (not sure if this captures all of the functionality): `startFileSystem` or `spawnFileSystem` (kind of like [`start_link`](https://hexdocs.pm/elixir/Supervisor.html#start_link/2) or [`spawn_link`](https://hexdocs.pm/elixir/Kernel.html#spawn_link/3))

Reply:
> Was chatting with @jessmartin about this. What would you think about `bootstrapFileSystem` instead of `installFileSystem`?

Technically speaking `establish` is more correct, but I like bootstrap as well 👍